examples/e2e-flow-github-auth-values.yaml

```diff
@@ -0,0 +1,108 @@
+global:
+  database:
+    mongodb:
+      host: bmrg-mongodb
+      secretName: bmrg-mongodb
+  ingress:
+    enabled: true
+    root: /poc-standalone/flow
+    host: ocp2.cloud.boomerangplatform.net
+    class: "nginx"
+    annotationsPrefix: "ingress.kubernetes.io"
+    tlsSecretName: ocp-secret-tls
+  auth:
+    enabled: true
+    mode: oauth
+    serviceName: flow-auth2proxy
+    namespace: poc-standalone
+    prefix: /poc-standalone/oauth
+    tokenURL: http://flow-auth2proxy.poc-standalone.svc.cluster.local/auth/token/validate/external
+    signOutRedirect: /poc-standalone/flow/apps/flow
+
+nats:
+  enable: false
+
+services:
+  controller:
+    image:
+      repository: /flow-service-controller
+      tag: 6.1.3
+    replicaCount: 1
+    service:
+      type: ClusterIP
+    zone: semi-trusted
+  listener:
+    image:
+      repository: /flow-service-listener
+      tag: 1.7.2
+    replicaCount: 1
+    service:
+      type: ClusterIP
+    zone: untrusted
+  workflow:
+    image:
+      repository: /flow-service-workflow
+      tag: 3.0.8
+    replicaCount: 1
+    service:
+      type: ClusterIP
+    zone: untrusted
```

Comment:
Do you think for the example we leave the versions out? that way we only have to keep them updated in the main values.yaml which is hard enough to keep in sync.

Reply:
Totally agree with you, I missed that part.
I'll remove the image tags.

Reply:
Thanks @morarucostel . Also potentially the `e2e` in the file title.